rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.59"
+channel = "1.60"
```

Comment:
I found that MSRV notification on `README.md` is still remaining 1.54.
It seems to be better to update it too 🙏